src/dgc_container.rs

```diff
@@ -12,17 +12,18 @@ const CERTS: i64 = -260;
 
 #[derive(Serialize, Debug, Clone, PartialEq)]
 pub struct DgcCertContainer {
-    #[serde(alias = "1")]
-    #[serde(rename(serialize = "1"))]
+    #[serde(alias = "1", rename(serialize = "1"))]
```

Comment:
`alias` shouldn't be necessary if `rename` is already available, doesn't it? Same thing for the other occurrences, obviously.